### DISCUSSION TITLE
iModel - create and maintaine

### DISCUSSION THREAD

**Discussion by thbnVD** [iTwin/itwinjs-core#5551] [Q&A]
We are looking for the ability to create and maintaine iModels.
Currently we use powershell, but it is not ideal do to token requirements.

According to the API, we can create iModels – but it does not seem to be possible to add a new connections and define schedule (manual/daily/weekly)  ?
Additionally, when creating a new named version, I would like to make sure, that no connections are running – is it possible to get the status of all connections for an iModel?

**Comment by thbnVD** [comment]
I found the "Synchronization" API, which i will look into.

**Reply by pratikshan85** [reply]
@thbnVD Please go through the below link for Synchronization API in detail.

https://developer.bentley.com/apis/synchronization

**Reply by thbnVD** [reply]
Thanks.
We are using ProjectWise to store all our designfiles - i can see an example for creating ManifestConnection to Azure blob and SharePoint. But nothing regarding ProjectWise, am i missing something, or is ManifestConnections to ProjectWise not supported out-of-the-box ?
